src/evaluate/__init__.py

```diff
@@ -26,6 +26,7 @@
 
 del version
 
+from .hub import *
```

Comment:
can we load the functions explicitly, makes tracking a bit easier and avoids accidentally expose private functions